JavaErickHernandez/Day3/ArrayLists.java

```diff
@@ -0,0 +1,93 @@
+package bootcamp;
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.*;;
+
+public class ArrayLists {
+	static ArrayList<number> numbers = new ArrayList<number>();
+	static String numberOfRows = "Enter the number of Rows: ";
```

Comment:
What is the reason for not putting these variables as "final"? if you put "final" you avoid to any other person can change this variables in other par of the code.

Reply:
I put them static in case I want to modify them in other part of the code hahaha, but it´s okay, I will change it